src/app/zap-templates/zcl/data-model/silabs/types.xml

```diff
@@ -251,12 +252,14 @@ limitations under the License.
     <item name="AboveTarget" value="0x2"/>
   </enum>
   <enum name="OccupancySensorType" type="ENUM8">
+    <cluster code="0x0406" />
     <item name="PIR" value="0x0"/>
     <item name="Ultrasonic" value="0x1"/>
     <item name="pirAndUltrasonic" value="0x2"/>
     <item name="physicalContact" value="0x3"/>
```

Comment:
These two names do not match the spec, for what it's worth.

Reply:
My spec copy uses `Description` only and does not seem to have an actual Name. Are there rules in naming? If one word, it seems like description == name, however what about cases when spaces exist? 

I would prefer my spec to give names as well for consistency.

Reply:
Yes, this should be an actual enum type in the spec, following the Discoball template, which would have a Name column, etc.  And then we would use the Name here.  But the above is not matching the closest we have to a spec-defined name for this.

As in, the spec here is broken, we should fix it, and then align to it, before we start exposing these things.

Reply:
Sorry I don't know where to make changes to the spec, I am not even sure I would have access to do so (please correct me if I'm wrong, and point me in the direction of the spec :) )
I have updated the XML files to match what is in the 1.0 spec